redshift_sqlalchemy/models.py

```diff
@@ -0,0 +1,55 @@
+"""
+System Tables and Views
+
+Amazon Redshift has many system tables and views that contain information
+about how the system is functioning. This module contains SQLAlchemy
+Declarative models for these tables.
+"""
+
+
+import sqlalchemy as sa
+from sqlalchemy.ext import declarative
+
+
+Base = declarative.declarative_base()
+
+
+class STLUnloadLog(Base):
+    """
+    Records the details for an unload operation.
+
+    STL_UNLOAD_LOG records one row for each file created by an UNLOAD
+    statement. For example, if an UNLOAD creates 12 files, STL_UNLOAD_LOG will
+    contain 12 corresponding rows.
+
+    This table is visible to all users.
+    """
+
+    __tablename__ = 'STL_UNLOAD_LOG'
+
+    user_id = sa.Column('userid', sa.Integer())
+    """ID of the user who generated the entry."""
+
+    query_id = sa.Column('query', sa.Integer(), primary_key=True)
+    """ID for the transaction."""
```

Comment:
I'm guessing it's a hard requirement that a declarative base class has a primary key?

The combination of `query_id` and `path` seems like a reasonable guarantee of uniqueness, but I'm generally uncomfortable with the idea of having to impose a primary key where none exists in the actual schema.